typescript/forms.d.ts

```diff
@@ -1225,3 +1221,11 @@ export type FormElementLookupSearchResponse = {
 export type FormElementOptionSetSearchResponse = {
   formElementOptionSets: FormElementOptionSet[]
 } & BaseSearchResult
+
+export type FormVersion = {
+  id: number
+  formId: number
+  form: Form
```

Comment:
This should prob omit a few things omitted

Reply:
> "omit a few things omitted"

 Vintage Matt.

Reply:
why omit anything?

Reply:
i am obsessed with removing duplicated data 🤷 I will not die on this hill.

Reply:
If we are not going to omit anything, do we need to store `createdAt`? Wouldn't that be `updatedAt` on the form? Assuming we are not going to store the current version in the `FormVersion` table...